### PR TITLE
[RC1 ch3]: html to md

### PR DESCRIPTION
**Pull request by CsatariGergely**

### CODE REVIEW

File: doc/ref_cert/lfn/chapters/chapter03.md [hunk 1]
```diff
@@ -474,20 +475,23 @@ integrated in
...
-| .\*RemoteStackTest.test_stack_create_with_cloud_credential | https://gerrit.opnfv.org/gerrit/c/functest/+/69926 |
-| .\*scenario.test_aodh_alarm                                | aodh                                               |
-| .\*tests.scenario.test_autoscaling_lb                      | lbaas                                              |
-| .\*scenario.test_autoscaling_lbv2                          | lbaasv2                                            |
-| .\*scenario.test_server_software_config                    | https://gerrit.opnfv.org/gerrit/c/functest/+/69926 |
-| .\*test_volumes.VolumeBackupRestoreIntegrationTest         | https://gerrit.opnfv.org/gerrit/c/functest/+/69931 |
-| .\*scenario.test_octavia_lbaas                             | octavia                                            |
-| .\*scenario.test_server_cfn_init                           | https://gerrit.opnfv.org/gerrit/c/functest/+/70004 |
+| test rejection regular expressions                                | reasons                                            |
+|-------------------------------------------------------------------|----------------------------------------------------|
```
Comment (tomkivlin) [inline doc/ref_cert/lfn/chapters/chapter03.md:482]: A minor point, but you only need three dashes to create the table structure, i.e. `|---|---|` would result in the same table.

**Comment by tomkivlin** [comment]
Suggest modifying the PR title to reflect that multiple workstreams and documents are affected, not just RC1 Ch3.

File: doc/ref_arch/openstack/chapters/chapter05.md [hunk 2]
```diff
@@ -217,6 +217,7 @@ The exhaustive list of extensions is available at https://docs.openstack.org/api
 | cold_migration           | X             |
 | console_output           | X             |
 | disk_config              | X             |
+| instance_password        | X             |
```
Comment (deebhatia) [inline doc/ref_arch/openstack/chapters/chapter05.md:220]: This file changeset is different that what PR indicates. Is it an intended change to be cherry-picked?

File: doc/ref_cert/lfn/chapters/chapter03.md (hunk 1 above)
Comment (CsatariGergely) [reply doc/ref_cert/lfn/chapters/chapter03.md]: I agree, but this change is about changing the links and also there are some of us who would like to keep the tables "aligned" on some level in their source format.

File: doc/ref_arch/openstack/chapters/chapter05.md (hunk 2 above)
Comment (CsatariGergely) [reply doc/ref_arch/openstack/chapters/chapter05.md]: Sorry I;ve selected the wrong base branch. Now it should be okay.

File: doc/ref_cert/lfn/chapters/chapter03.md (hunk 1 above)
Comment (tomkivlin) [reply doc/ref_cert/lfn/chapters/chapter03.md]: no problem - looked like a bunch of different files had been changed, including this line - but you've sorted that now. Resolved.

**Pull request merged**